### ISSUE TITLE
Can you share the finetune.sh and pretrain.sh to train TinyLLaVA-1.5B?

### ISSUE THREAD

**Issue by fyting** [DLCV-BUAA/TinyLLaVABench#42]
Can you share the finetune.sh and pretrain.sh to train TinyLLaVA-1.5B?

**Comment by baichuanzhou** [comment]
change model's name from bczhou/TinyLLaVA-3.1B to TinyLLaVA-1.5B will do

**Comment by FlyingWorkshop** [comment]
I've tried doing something similar, but I've gotten `WARNING: tokenization mismatch: 69 vs. 76. (ignored)` warning

**Comment by letitiabanana** [comment]
I also see this warning when tring to run qwen2.5 as llm "WARNING: tokenization mismatch: 509 vs. 513. (ignored) " Can you please share the reason for the warning in make_labels? Thanks a lot!

def make_labels(self, input_ids, prompt, tokenizer):
        labels = copy.deepcopy(input_ids)
        sep, eos_token = self.separator.apply()
        total_len = int(labels.ne(tokenizer.pad_token_id).sum())
        if tokenizer.pad_token_id == tokenizer.eos_token_id:
            total_len += prompt.count(eos_token)
        rounds = prompt.split(eos_token)
        eos_token_length = len(tokenizer.encode(eos_token))
        labels, cur_len = self._make_masks(labels, tokenizer, sep, eos_token_length, rounds)
        if cur_len < tokenizer.model_max_length:
            import time
            if cur_len != total_len:
                print(
                    f"WARNING: tokenization mismatch: {cur_len} vs. {total_len}."
                    f" (ignored)"
                )
                print("number of rounds: ", len(rounds) - 1)
                print("rounds: ", rounds[:-1])
                print("prompt: ", prompt)
                print(labels)
                print(input_ids)
                time.sleep(5)
                labels[:] = IGNORE_INDEX
        return labels

**Comment by letitiabanana** [comment]
I figured out why tokenization mismatch: 509 vs. 513.. It is mostly because the new llm you are trying to incorporate into Tinyllava has different tokenizer_config
